notebooks/query-language-object-based-dialect/01.01.quantifier_capabilities.ipynb

```diff
@@ -0,0 +1,842 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b0f5e933",
+   "metadata": {},
+   "source": [
+    "# Hatchet Query Language\n",
+    "\n",
+    "\n",
+    "This notebook explores [Hatchet](https://github.com/LLNL/hatchet) queries from it's [**Object-based Dialect**](https://hatchet.readthedocs.io/en/latest/query_lang.html) within **Category 1: Quantifier Capabilities**. Below, Fig. 1 provides an overview of all the categories within the Hatcher Query Language.   "
```

Comment:
I feel the statement introduces too many concepts for a user who is beginning to use the notebook. Consider simplifying the sentence to only talk about the Object-based dialect and also add a brief definition of object-based dialect instead of pointing it to `readthedocs`. I am also not sure if we should enlist all categories as a table if we are going to have separate notebooks for each category.

Consider something like,

"This notebook demonstrates the functionalities of Hatchet-query language. In particular, we will explore queries that follow the **Object-based dialect**. Hatchet-query language also supports 5 different categories as shown in Fig. 1.